stdlib/multiprocessing/pool.pyi

```diff
@@ -26,9 +23,18 @@ class ApplyResult(Generic[_T]):
 # alias created during issue #17805
 AsyncResult = ApplyResult
 
-class MapResult(ApplyResult[List[_T]]): ...
+class MapResult(ApplyResult[List[_T]]):
+    def __init__(
+        self,
+        pool: Pool,
+        chunksize: int,
+        length: int,
+        callback: Optional[Callable[[List[_T]], None]],
+        error_callback: Optional[Callable[[BaseException], None]],
+    ) -> None: ...
```

Comment:
Same here. Also, please use `| None` and built-in generics, at least for new annotations.